src/templates/styles/locations.scss

```diff
@@ -43,10 +43,12 @@
   padding: 1.5rem;
   display: grid;
   grid-template-columns: 1fr;
+  background-color: var(--primary);
+  color: var(--text-alt);
```

Comment:
DO NOT NEED TO CHANGE

curious if resetting the standard font color here to something like  `--textNormal: var(--backgroundAlt)` would fix the problem without also needing to restyle the button's color explicitally